watchdog-config/extra-sql-queries.yml

```diff
@@ -75,3 +75,59 @@ users_with_insificient_disk:
         usage: "LABEL"
     - cht_instance:
         usage: "LABEL"
+
+days_since_successful_purge:
+  query: |
+    SELECT 
+      purge.partner_name,
+      replace(monitoring_urls.url, 'https://', '') as cht_instance,
+      now()::date - MAX(completion_date) FILTER(WHERE error IS NULL)::date AS "days"
+    FROM
+      public.app_monitoring_purge_logs as purge
+    LEFT JOIN
+      monitoring_urls ON purge.partner_name=monitoring_urls.partner_name
+    GROUP BY
+      purge.partner_name,
+      cht_instance
+  metrics:
+    - days:
+        usage: "COUNTER"
+    - partner_name:
+        usage: "LABEL"
+    - cht_instance:
+        usage: "LABEL"
+
+form_completions_60_days:
+  query: |
+    SELECT
+      form_name AS form_name,
+      sum(load_count - complete_count) AS incomplete_count,
+      sum(complete_count) AS complete_count,
+      AVG(complete_count/load_count::float) AS complete_percentage,
```

Comment:
IMHO we should not record this metric.  We can easily calculate this in Grafana so having it here seems like a small waste of Postgres processing and Prometheus storage.

Reply:
fair! removed in a629e16